inc/labs/class.llms.lab.action.manager.php

```diff
@@ -248,6 +248,11 @@ public function setup_hooks() {
 						'priority' => 20,
 						'title' => __( 'Course Navigation Tiles', 'lifterlms-labs' ),
 					),
+					'lifterlms_template_single_favorite' => array(
```

Comment:
I think that this one should be conditionally added base on whether the feature is available in the core.
Otherwise we just need to raise the minimum LifterLMS version this plugin requires.